src/rules/speech_control.ts

```diff
@@ -1178,4 +1178,36 @@ export function initRules_bc_speech_control() {
 			}, ModuleCategory.Rules);
 		}
 	});
+
+	registerRule("speech_garble_while_talking", {
+		name: "Force garbled speech",
+		type: RuleType.Speech,
+		loggable: false,
+		shortDescription: "force PLAYER_NAME to talk as if they were gagged",
+		longDescription: "This rule forces PLAYER_NAME to talk as if they were gagged, automatically garbling all of their speech. This rule does not affect OOC or, if \"Garble whispers while gagged\" is disabled, whispers.",
```

Comment:
Suggestion from us that is a bit more explicit: (also using backticks to avoid need to escape)
```suggestion
		longDescription: `This rule forces PLAYER_NAME to talk as if they were gagged, automatically garbling all of their speech. This rule does not affect OOC. This rule only affects whispers if the rule "Garble whispers while gagged" is also in effect.`,
```